service/DevDocs/ServiceClient.md

```diff
@@ -0,0 +1,30 @@
+# ServiceClient Requirements
+
+## Overview
+
+ServiceClient class is used by the Service client callback delegate to prepare the callback results.
+
+### References
```

Comment:
@tameraw @jasmineymlo - I see 3 dev-docs having this References but none actually filling it. What should go here? (We can still have a template checked in containing it, explaining what each of these fields mean.)

Maybe we can add a link to the AMQP SendAsync API that we're passing this through? If not, we can just delete it.

Reply:
Could probably point a link to the public spec.

---
In reply to: [140533683](https://github.com/Azure/azure-iot-sdk-csharp/pull/160#discussion_r140533683) [](ancestors = 140533683)